js/gsce-custom.js

```diff
@@ -0,0 +1,176 @@
+const DOMAIN_URL_LENGTH = "https://mechanicalrock.github.io/".length;
```

Comment:
This file doesn't have any tests.

Is this boilerplate from Google?

Untested code is a source of risk and should probably be deleted.

Reply:
This is not boilerplate from Google. This is custom JS written by me in order to supplement some of the styling that Google enforces on the search results. With regards to testing, I need some help with that as I do not understand how we could incorporate a JS test runner with Jekyll

Reply:
A quick google suggests just to run them on the side using jest/cypress/whatever.

I guess my questions would be:
* What should we test?
* Where are the risks? (what might break)

Before deciding on the best approach to writing the tests.

We can incorporate them as a GitHub actions hook, prior to merge.

Reply:
I have now added Jest unit tests for both custom js files